packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -389,6 +391,8 @@ export class ARRenderer extends EventDispatcher {
     camera.near = 0.1;
     camera.far = 100;
 
+    (this.threeRenderer.xr as any).updateCamera(camera);
```

Comment:
@Mugen87 I'm calling updateCamera on every frame already right here, but indeed, if I turn `cameraAutoUpdate` back to `true`, the problem goes away. Interesting...

Reply:
Actually, I take that back. This seems to be working now with `cameraAutoUpdate = false` after all. My best guess is I confused myself by forgetting when I had dev tools open, which introduces it's own frame delay. My apologies for the false alarm!